draft-packages/menu/package.json

```diff
@@ -40,9 +40,11 @@
     "react-popper": "^2.2.4"
   },
   "devDependencies": {
+    "@kaizen/draft-divider": "^1.3.0",
     "rimraf": "^3.0.2"
   },
   "peerDependencies": {
+    "@kaizen/draft-divider": "^1.3.0",
```

Comment:
I think we're leaning towards using normal dependencies for most sub-components, and peer-dependencies just for design tokens and react, where you really don't want multiple versions.

The effort to install a package goes way up with peer dependencies and it can get very messy very quickly, especially when you have multiple incompatible requirements for a package. A bit of potential duplication is worth the trade-off